packages/common/src/styles/theme.ts

```diff
@@ -122,15 +113,14 @@ const themeOptions = {
     },
   },
   palette: {
-    darkGrey: { main: '#555770' },
+    outline: { main: '#555770' },
     divider: '#eaeaea',
     error: { main: '#e63535' },
-    lightGrey: '#c7c9d9',
-    midGrey: '#8f90a6',
+    gray: { main: '#8f90a6', light: '#c7c9d9', dark: '#555770' },
```

Comment:
I just think it is neater to have an object instead of `lightGrey` `midGrey` etc. Would prefer not to have `gray` and `paleGrey` but 🤷